contracts/TestToken.sol

```diff
@@ -1,15 +1,15 @@
 pragma solidity 0.6.6;
 
 import "openzeppelin-solidity/contracts/token/ERC20/ERC20.sol";
-import "openzeppelin-solidity/contracts/access/Ownable.sol";
+import "./Ownable.sol";
```

Comment:
there were 2 Ownable contracts, so I converted this to use ours. OpenZeppelin's Ownable doesn't suit our needs because it cant be used with DU2 factory pattern (owner must be set in constructor).